packages/react-select/src/components/Select/useSelectStyles.ts

```diff
@@ -12,36 +13,211 @@ export const selectClassNames: SlotClassNames<SelectSlots> = {
   icon: 'fui-Select__icon',
 };
 
-const useStyles = makeStyles({
-  wrapper: {
-    // TODO: add styles
+// TODO(sharing) use theme values once available
+const horizontalSpacing = {
+  xxs: '2px',
+  xs: '4px',
+  sNudge: '6px',
+  s: '8px',
+  mNudge: '10px',
+  m: '12px',
+};
+const contentSizes = {
+  // TODO: borrowed this from Input, should be in the theme somewhere?
```

Comment:
aren't these styles exported by `react-text` ?

Reply:
https://github.com/microsoft/fluentui/blob/3651218a1e8069fd6fcb095ef5d30139f8c01ff4/packages/react-text/src/typographyStyles/typographyStyles.ts#L13

Reply:
from chatting with @ling1726 -- I'm going to update these to pull from `react-text`, and I'll create a separate PR to update in Input as well 👍

Reply:
@andrefcdias  is it intentional that only the text components are exported but not the typography styles ? this seems like exactly the case where we would need the styles to avoid duplication

Reply:
Actually, since `typographyStyles` are not currently exported from `react-text`, @ling1726 how would you feel about keeping this as-is for now and updating both Select and Input when that gets resolved?

Reply:
Outcome from tech sync was not to export until needed due to bundle size concerns. Let me know if you want me to open a PR with the exports or feel free to open one and tag me.

Reply:
The main issue with duplicating text styles internally is that it would increase bundle size for the whole library since these styles will not be deduplicated

Reply:
I've added this to tech sync we will discuss offline how to deal with the duplicated styles

Reply:
Created this issue: https://github.com/microsoft/fluentui/issues/22231